doc/neomake.txt

```diff
@@ -31,13 +31,13 @@ fantastic vim plugins such as syntastic and dispatch.
 ==============================================================================
 2. Commands                                                   *neomake-commands*
 
-                                                         *:Neomake* *:NeomakeFile*
+                                                       *:Neomake* *:NeomakeFile*
```

Comment:
Well, this is wrong when being concealed.. the `*` will be removed and then this is less indented than the line above.
Or is this intentional?
Guess it might be good as-is for now (since the width is too wide anyway), see https://github.com/neomake/neomake/issues/600.